src/pages/Search.tsx

```diff
@@ -25,7 +25,7 @@ export const Search = () => {
         <Results />
         <MapBox />
         <SearchBox>
-          <SearchForm />
+          <SearchForm closed={true} />
```

Comment:
is it correct that this is always `closed` ?

Reply:
yes